docs/sources/nodes.md

```diff
@@ -111,7 +112,7 @@ spec:
         - --domain-filter=external-dns-test.my-org.com
         - --aws-zone-type=public
         - --registry=txt
-        - --fqdn-template={{.Name}}.external-dns-test.my-org.com
+        - --fqdn-template={{.Name | replace "." "-"}}.external-dns-test.my-org.com
```

Comment:
I never tried this `fqdn-template`, could you create a page or a section with bit more information about this `fqdn-template` and it's use cases and advantages?

I can see an example for raw kubernetes manifest. How this behave in helm chart? as the syntax is very similar. I would assume helm generation will fail, am I correct?